www/src/views/shared/collapsible.js

```diff
@@ -1,83 +1,75 @@
 /** @jsx jsx */
 import { jsx } from "theme-ui"
-import { Component } from "react"
+import { useState } from "react"
 import { FaAngleDown, FaAngleUp } from "react-icons/fa"
 
-class Collapsible extends Component {
-  state = {
-    collapsed: false,
-  }
+import { Flex } from "theme-ui"
 
-  handleClick = () => {
-    this.setState({ collapsed: !this.state.collapsed })
-  }
+export default function Collapsible(props) {
+  const { heading, fixed, children } = props
+  const [collapsed, setCollapsed] = useState(false)
 
-  render() {
-    const { heading, fixed, children } = this.props
-    const { collapsed } = this.state
+  function toggleCollapsible() {
+    setCollapsed(!collapsed)
+  }
 
-    return (
-      <div
+  return (
+    <div
+      sx={{
+        borderBottom: t => (collapsed ? 0 : `1px solid ${t.colors.ui.border}`),
+        display: collapsed ? false : `flex`,
+        flex: collapsed ? `0 0 auto` : `1 1 auto`,
+        minHeight: fixed ? `${fixed}px` : `initial`,
+        maxHeight: fixed ? `${fixed}px` : `initial`,
+        flexBasis: 0,
+        overflowY: collapsed ? false : `auto`,
+      }}
+    >
+      <Flex
         sx={{
-          borderBottom: t =>
-            collapsed ? 0 : `1px solid ${t.colors.ui.border}`,
-          display: collapsed ? false : `flex`,
-          flex: collapsed ? `0 0 auto` : `1 1 auto`,
-          minHeight: fixed ? `${fixed}px` : `initial`,
-          maxHeight: fixed ? `${fixed}px` : `initial`,
-          flexBasis: 0,
-          overflowY: collapsed ? false : `auto`,
+          flexDirection: `column`,
+          minHeight: `100%`,
+          width: `100%`,
         }}
       >
-        <div
-          css={{
+        <button
+          sx={{
+            alignItems: `center`,
+            color: `textMuted`,
+            cursor: `pointer`,
             display: `flex`,
-            flexDirection: `column`,
-            minHeight: `100%`,
-            width: `100%`,
+            flexShrink: 0,
+            fontWeight: `body`,
+            fontSize: 1,
+            my: 6,
+            mr: 4,
+            p: 0,
+            letterSpacing: `tracked`,
+            textTransform: `uppercase`,
+            background: `none`,
+            border: `none`,
+            "&:hover": {
+              color: `gatsby`,
+            },
           }}
+          aria-expanded={!collapsed}
+          onClick={toggleCollapsible}
```

Comment:
```suggestion
          onClick={() => setCollapsed(collapsed => !collapsed)}
```

Reply:
It was like this earlier but after suggestion from @muescha, I converted it to function declaration and called it from here.

Reply:
Even I don't think passing arrow function directly to onClick would be a good idea

Reply:
It's definitely a matter of preference, but since it's a one liner, I think it's clearer to put the handler in same place. If you search for `onClick=` in the codebase you'll see we inline the handlers most of the time.

At the very least, we could use [functional updates](https://reactjs.org/docs/hooks-reference.html#functional-updates) version.

Reply:
Okay, I will update it give me few minutes